### DISCUSSION TITLE
Where is trained model?

### DISCUSSION THREAD

**Discussion by Holmes2002** [yoshitomo-matsubara/torchdistill#261] [Q&A] [answered]
Hi, I had asked you location of trained model, I 'm training it and I don't see . Is Training end I will have or I don't create folder for it ?

**Comment by Holmes2002** [comment]
Ok, I checked again. I have to create folder fisrtly !

**Comment by yoshitomo-matsubara** [comment, marked as answer]
Hi @Holmes2002 

You don't have to create a ckpt folder by yourself.
If the folder doesn't exist, the script will create it for you during training
